tasks/test-thoth-github-action.yaml

```
@@ -0,0 +1,112 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: test-thoth-github-action
+spec:
+  params:
+    - name: head_commit_hash
+      description: The hash of the last commit from the push event
+    - name: default_branch
+      description: The git repository default branch
+    - name: ssh_url
+      description: The git repository ssh url
```

This ssh_url is pointing to the ssh URL of the repo it was triggered from.

our case is little different right!
we want to trigger the changes in `test` based on the trigger i.e release of thoth-github-action.

This could be achieved by defining these detail in the configuration file.
how about we do this in .aicoe-ci.yaml
```

thoth-github-action:
   test-repo: ssh_url for the test repo
   test-branch: branch to push these changes
```
```suggestion
    - name: pr_branch
      description: The git repository default branch
      
      
    - name: ssh_url
      description: The git repository ssh url
```
```suggestion
    - name: default_branch
      description: The git repository default branch
      default: test
    - name: ssh_url
      description: The git repository ssh URL
      default: "<git@github ...>"
      
```
however i feel as this is special case , such task wont be used by any other project
so lets hard code.